src/awst_build/eb/arc4/tuple.ts

```diff
@@ -23,7 +23,7 @@ export class Arc4TupleClassBuilder extends ClassBuilder {
       genericTypeArgs: 1,
       funcName: this.typeDescription,
       callLocation: sourceLocation,
-      argSpec: (a) => [a.required(), ...args.slice(1).map(() => a.required())],
+      argSpec: (a) => [...args.map(() => a.required())],
```

Comment:
Again the original code here was deliberate to prevent empty tuples

Reply:
It has been reverted.